plugins/system_upgrade.py

```diff
@@ -72,6 +72,20 @@ def reboot():
     Popen(["systemctl", "reboot"])
 
 
+def get_url_from_os_release():
+    key = "DOCUMENTATION_URL="
+    for path in ["/etc/os-release", "/usr/lib/os-release"]:
+        try:
+            with open(path) as release_file:
+                for line in release_file:
+                    line = line.strip()
+                    if line.startswith(key):
+                        return line.lstrip(key + '"').rstrip('"')
```

Comment:
Note that [`lstrip`](https://docs.python.org/3/library/stdtypes.html#str.lstrip) takes a *set of characters*, not a prefix to be removed.
```
>>> "DOCUMENTATION_URL=ABC".lstrip('"DOCUMENTATION_URL=')
'BC'
```